packages/keys/src/key-pair.ts

```diff
@@ -1,7 +1,10 @@
 import { assertKeyGenerationIsAvailable } from '@solana/assertions';
-import { SOLANA_ERROR__KEYS__INVALID_KEY_PAIR_BYTE_LENGTH, SolanaError } from '@solana/errors';
+import { SOLANA_ERROR__KEYS__INVALID_KEY_PAIR_BYTE_LENGTH, SolanaError, SOLANA_ERROR__KEYS__PUBLIC_KEY_MUST_MATCH_PRIVATE_KEY } from '@solana/errors';
 
 import { createPrivateKeyFromBytes } from './private-key';
+import { verifySignature, signBytes } from './signatures';
+
+const TEST_DATA = new Uint8Array([1, 2, 3, 4, 5]);
 
 export async function generateKeyPair(): Promise<CryptoKeyPair> {
     await assertKeyGenerationIsAvailable();
```

Comment:
To generate a test message randomly each time, we'll need to assert that the PRNG is available (that `crypto.getRandomValues` is a function`. Can you add that, and a related error code, to `@solana/assertions`.

```suggestion
    await Promise.all([
        assertKeyGenerationIsAvailable(),
        assertPRNGIsAvailable(), // <-- add this to `@solana/assertions` & `@solana/errors`
    ]);
```

Reply:
I think it would make more sense to put `assertPRNGIsAvailable()` in `createKeyPairFromBytes` than `generateKeyPair` right?

Reply:
Oh, of course. Oops; I left my comment in the wrong spot!